include/sbuf.h

```diff
@@ -42,6 +42,8 @@ typedef enum {
 
 struct tls;
 
+extern int client_accept_sslmode;
```

Comment:
The use of this variable isn't clear to me. Please add a comment somewhere if appropriate.

Reply:
done